infra/main.bicep

```diff
@@ -442,6 +442,7 @@ module application './modules/application-resources.bicep' = {
     frontDoorSettings: frontdoor.outputs.settings
 
     // Settings
+    principalId: principalId
```

Comment:
can you remove this by using the `principalId` in the DeploymentSettings object?

Reply:
for sure - didn't realize it was there!